examples/2-query/webpack.config.js

```diff
@@ -11,5 +11,12 @@ module.exports = {
   devServer: {
     open: true,
     contentBase: path.resolve(__dirname, "public")
+  },
+  resolve: {
+   alias: {
```

Comment:
How would you feel about opting for the module resolution we have setup in `3-mutation`, where we look in the project root before checking local `node_modules`?

Reply:
I don't know much about Webpack configuration, when I tried to configure the resolve in `2-query` the same way it was done in `3-mutation`, it gave me some errors.
But I believe that it would be good to do it like in `3-mutation`